test/tests.js

```diff
@@ -451,6 +451,19 @@ QUnit.test('Call to read all cookies with mixed json', function (assert) {
 	assert.deepEqual(Cookies.get(), { c: '{"foo":"bar"}', c2: 'v' }, 'returns unparsed cookies');
 });
 
+QUnit.test('Cookies with escaped quotes in json', function (assert) {
+	Cookies.withConverter({
+		read: function (value) {
+			return value;
+		},
+		write: function (value) {
+			return value;
+		}
```

Comment:
Why are the converters necessary?

Can you just do:

```javascript
Cookies.set('c', '"{ \\"foo\\": \\"bar\\" }"');
```

Reply:
By default the quotes will get escaped which is not the value I want to test. I want the final value to be exactly `"{\"foo\":\"bar\"}"`

If I update the code like you said it fails:
```
>> Message: returns unparsed cookie with enclosing quotes removed
>> Actual: "\"{ \\\"foo\\\": \\\"bar\\\" }\""
>> Expected: "{ \\\"foo\\\": \\\"bar\\\" }"
```

Reply:
Also, after removing the converters, the first test case passes but it's not actually testing the case that I am fixing with my change.

Reply:
Oh, I see, now that makes sense. You want to test the raw converters and the raw `getJSON()`. Thanks for the clarification.